login.php

```diff
@@ -4,157 +4,76 @@
-          <div class="input-group mb-3">
-            <input type="password" class="form-control" name="password" required placeholder="Password">
-            <div class="input-group-append">
-              <div class="input-group-text">
-                <span class="fas fa-lock" onclick="togglePasswordVisibility()"></span>
-                <span class="fas fa-eye" style="display:none;" onclick="togglePasswordVisibility()"></span>
-              </div>
+<body>
+    <h1 id="page-title" class="text-center">Login Page</h1>
+    <hr id="title_hr" class="mx-auto">
+    <div id="login-wrapper">
+        <div class="text-muted"><small><em>Please fill in all the required fields</em></small></div>
+        <?php if (isset($error) && !empty($error)): ?>
+            <div class="message-error"><?= $error ?></div>
+        <?php endif; ?>
+        <?php if (isset($_SESSION['msg']['success']) && !empty($_SESSION['msg']['success'])): ?>
+        <div class="message-success">
+            <?php 
+            echo $_SESSION['msg']['success'];
+            unset($_SESSION['msg']);
+            ?>
+        </div>  
+        <?php endif; ?>
+        <form action="" method="POST">
+            <div class="input-field">
+                <label for="email" class="input-label">Email</label>
+                <input type="email" id="email" name="email" value="<?= htmlspecialchars($_POST['email'] ?? "", ENT_QUOTES) ?>" required="required">
             </div>
-          </div>
-          <div class="form-group mb-3">
-            <label for="">Login As</label>
-            <select name="login" id="" class="custom-select custom-select-sm">
-              <option value="3">Student</option>
-              <option value="2">Faculty</option>
-              <option value="1">Admin</option>
-            </select>
-          </div>
-          <div class="row">
-            <div class="col-8">
-              <div class="icheck-primary">
-                <input type="checkbox" id="remember">
-                <label for="remember">
-                  Remember Me
-                </label>
-              </div>
+            <div class="input-field">
+                <label for="password" class="input-label">Password</label>
+                <input type="password" id="password" name="password" value="<?= htmlspecialchars($_POST['password'] ?? "", ENT_QUOTES) ?>" required="required">
             </div>
-            <!-- /.col -->
-            <div class="col-4">
-              <button type="submit" class="btn btn-primary btn-block">Sign In</button>
+            <div class="input-field">
+                <a href="forgot-password.php" tabindex="-1"><small><strong>Forgot Password?</strong></small></a>
             </div>
-            <p class="mb-1">
-              Don't have an account? <a href="signup.php" class="text-center">Sign up</a>
-            </p>
-
-            <!-- /.col -->
-          </div>
+            <button class="login-btn" type="submit">Login</button>
```

Comment:
Why are these changes coming up again? I thought you had modified in the previous PR. Please do a latest pull before raising PR